samples/c2pa/C2paPlayer/C2paMenu/C2paMenu.js

```diff
@@ -48,24 +54,60 @@ export var C2PAMenu = function () {
         //Functions to access the c2pa menu items from the c2pa manifest
         c2paItem: function (itemName, c2paStatus, compromisedRegions = []) {
             const verificationStatus = c2paStatus.verified;
-            let manifest = null;
+            let manifest, producer, socialMedia = null;
             try {
                 manifest = c2paStatus.details.video.manifest;
             } catch (error) {
                 console.error('[C2PA] Manifest does not exist');
             }
-
+            
             if (manifest != null && manifest['manifestStore'] != null) {
                 if (itemName == 'SIG_ISSUER') {
                     return manifest['manifestStore']['activeManifest'][
                         'signatureInfo'
                     ]['issuer'];
```

Comment:
Not sure if the 'issuer' info is always present in the manifest, otherwise, we can catch whether the value is null.

Reply:
I'll check!

Reply:
okay so issuer and claim generator are required in the manifest , so no check is needed

Reply:
Thanks for the info!